frontend/storage/configureStore.ts

```diff
@@ -2,6 +2,7 @@ import { createStore, Store } from 'redux';
 import { AppActions, AppStore } from './common';
 import { appReducer } from './reducers';
 import { Game } from './models/Game';
+const SuperMetroidCover = require('../assets/cover_super_metroid.jpg');
```

Comment:
I presume you couldn't use `import` because typescript complained?

You need to add definition file. File like [this](https://github.com/EvidentlyCube/chronic-temporal/blob/master/src/index.d.ts) that has to be included somewhere [like this](https://github.com/EvidentlyCube/chronic-temporal/blob/master/src/Bootstrap.ts#L2). Then you can import the images [like this](https://github.com/EvidentlyCube/chronic-temporal/blob/master/src/Initialization/queueAssets.ts#L4). Here is the relevant part of the [webpack config](https://github.com/EvidentlyCube/chronic-temporal/blob/master/webpack.config.js#L40) that this project uses

Reply:
This should ensure `SuperMetroidCover` will be detected to be a string and you won't have to mark things as `any` anywhere.

Relevant: if you ever have a situation where you get something that is `any` you should figure out its type as early as possible. The image URL WILL have to be a string, so there was no point making the `img` type to any, and then having to possibly reconcile it later.
Imagine a situation where you get something from the API in **one** place that's used in **five** places. If you reconcile the type immediately then you can use the value in those five places as is with strong type checking. If you wait until they are actually used you have to do the same work in 5 places.